src/lib/pages/workflow-history-feed.svelte

```diff
@@ -2,10 +2,40 @@
   import { timelineEvents } from '$lib/stores/events';
   import EventSummary from '$lib/components/event/event-summary.svelte';
   import { onDestroy } from 'svelte';
+  import { page } from '$app/stores';
+  import { fetchAllEvents } from '$lib/services/events-service';
+  import { eventFilterSort, type EventSortOrder } from '$lib/stores/event-view';
+  import { refresh } from '$lib/stores/workflow-run';
+
+  let events: CommonHistoryEvent[] = [];
+
+  $: namespace = $page.params.namespace;
+  $: workflowId = $page.params.workflow;
+  $: runId = $page.params.run;
```

Comment:
There are couple places where we could do this instead 🔽 
```suggestion
  $: ({ namespace, workflow: workflowId, run: runId } = $page.params);
```